mne_qt_browser/_pg_figure.py

```diff
@@ -1862,13 +1861,20 @@ def __init__(self, main):
         self.setLayout(layout)
         self.show(center=False)
 
-    def _chkbx_changed(self, label):
+    def _chkbx_changed(self, checked, label=None):
```

Comment:
Okay we got in CIs:
```
_________________________ test_plot_raw_selection[qt] __________________________
../mne-python/mne/viz/tests/test_raw.py:343: in test_plot_raw_selection
    fig._fake_click((0.5, 0.5), fig=chkbx)
mne_qt_browser/_pg_figure.py:4489: in _fake_click
    raise RuntimeError(f'There as been an {exc[0]} inside the Qt '
E   RuntimeError: There as been an <class 'TypeError'> inside the Qt event loop (look above for traceback).
----------------------------- Captured stderr call -----------------------------
Exceptions caught in Qt event loop:
________________________________________________________________________________
Traceback (most recent call last):
  File "/home/runner/work/mne-qt-browser/mne-qt-browser/mne_qt_browser/_pg_figure.py", line 2674, in call
    return meth_(*args_, **kwargs, **kwargs_)
TypeError: SelectionDialog._chkbx_changed() missing 1 required positional argument: 'checked'
```
So just to be safe let's go ahead and do:
```suggestion
    def _chkbx_changed(self, checked=True, label=None):
```

Reply:
Very weird that it failed only with PySide2.. That would mean there was a call with label as a keyword argument somewhere.